wandb/sdk/internal/internal_api.py

```diff
@@ -186,7 +186,7 @@ def app_url(self):
 
     @property
     def default_entity(self):
-        return self.settings("entity")
+        return self.viewer().get("entity")
```

Comment:
Why is this change needed? Does it impact other parts of the CLI that use this function?

Reply:
the previous `self.settings("entity")` (which i think was attributed to you?) returned empty for me. as far as i can tell this fn isn't used anywhere but launch_agent

Reply:
Oh, haha, I see. Okay, this looks good, matches the public api was well.